wcfsetup/install/files/acp/templates/articleList.tpl

```diff
@@ -21,7 +21,7 @@
 
 <header class="contentHeader">
 	<div class="contentHeaderTitle">
-		<h1 class="contentTitle">{lang}wcf.acp.article.list{/lang}</h1>
+		<h1 class="contentTitle">{lang}wcf.acp.article.list{/lang}  <span class="badge badgeInverse">{#$items}</span></h1>
```

Comment:
Superfluous whiespace between the title and the badge. Applies to some other changes too.